packages/react/src/summary-list/docs/overview.mdx

```diff
@@ -52,41 +52,50 @@ Common uses for this component are to implement a glossary or to display metadat
 
 ## Actions
 
-You can add actions to the end of a SummaryList by nesting `SummaryListItemAction` components.
+You can add actions to the end of a SummaryList by nesting `<SummaryListItemAction>`s and interactive components such as `<TextLink>`.
+
+Ensure each action’s accessible label is unique for screen reader users by including `<VisuallyHidden>` text that matches the `<SummaryListItemTerm>`’s content.
 
 ```jsx live
 <SummaryList>
 	<SummaryListItem>
 		<SummaryListItemTerm>First name</SummaryListItemTerm>
 		<SummaryListItemDescription>Will</SummaryListItemDescription>
 		<SummaryListItemAction>
-			<TextLink href="#">Change</TextLink>
+			<TextLink href="#">
+				Change <VisuallyHidden>first name</VisuallyHidden>
```

Comment:
I originally tried allowing `id` and `aria-describedby` but VoiceOver didn't pick it up. It was also a lot more cumbersome codewise